draft-ietf-quic-transport.md

```diff
@@ -593,37 +593,40 @@ The server populates the Destination Connection ID with the connection ID that
 the client included in the Source Connection ID of the Initial packet.
 
 The server includes a connection ID of its choice in the Source Connection ID
-field.  The client MUST use this connection ID in the Destination Connection ID
-of subsequent packets that it sends.
+field.  This value MUST not be equal to the Destination Connection ID field of
+the packet sent by the client.  The client MUST use this connection ID in the
+Destination Connection ID of subsequent packets that it sends.
 
-A Retry packet does not include a packet number and cannot be explicitly
-acknowledged by a client.
+A server MAY send Retry packets in response to Initial and 0-RTT packets.  A
+server can either discard or buffer 0-RTT packets that it receives.  A server
+can send multiple Retry packets as it receives Initial or 0-RTT packets.
 
-A server MUST NOT send a Retry in response to packets other than Initial
-or 0-RTT packets.  A server MAY choose to only send Retry in response to Initial
-packets and discard or buffer 0-RTT packets corresponding to unvalidated client
-addresses.
+A client MUST accept and process at most one Retry packet for each connection
```

Comment:
This MUST is a bit strong for me when thinking of man-on-the-side, especially since you don't respond to cases when the original dcid doesn't match, but it's Ok for now since mots is out of scope for v1.

Reply:
At some point it was suggested that a man-on-the-side defence could be to excercise the retry, but still watch out of an initial response within a timeout period. If an initial response arrives, the retry is cancelled. But I guess you could have bogus inital responses to valid retries ...